utils.js

```diff
@@ -32,7 +41,11 @@ async function describeTable({ dynamodb, name }) {
   }
 }
 
-async function updateTable({ dynamodb, name, attributeDefinitions }) {
+async function updateTable({
+  dynamodb,
+  name,
+  attributeDefinitions,
```

Comment:
After creating the table, can we update it with the new indexes? Or do we have to define indexes only on create? I don't remember dynamoDB limits on this 🤔

Reply:
You are correct. I was thinking of submitting another PR. Since I am in desperate need of this code, I was wondering if you could just merge this now, and I will create an additional PR to update the indexes.

Reply:
I have actually added in this PR now. I also updated the documentation.